source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -531,9 +532,22 @@ export class Template {
      * @returns A value representing the expanded template. The type of that value
      *   is dependent on the type of the original template payload passed to the constructor.
      */
-    expand(context: IEvaluationContext): any {
+    expand(context: IEvaluationContext, errorLog?: String[]): any {
```

Comment:
This logic could also be done with an optional boolean, but I wasn't sure if we wanted devs to have the flexibility to pass in an existing array.

It would also be simplified with an out parameter, but I wasn't sure how to do this in typescript.